js/main.js

```diff
@@ -0,0 +1,14 @@
+(function($) {
```

Comment:
Codacy found an issue: [Move the invocation into the parens that contain the function.](https://app.codacy.com/gh/BuildForSDGCohort2/team-040-frontend/pullRequest?prid=6093393)